src/main/java/lotto/domain/BuyLotto.java

```diff
@@ -0,0 +1,76 @@
+package lotto.domain;
+
+import lotto.domain.constant.LottoRank;
+
+import java.util.List;
+import java.util.Objects;
+
+public class BuyLotto {
+
+    private final List<Lotto> lottos;
+
+    public BuyLotto(Integer... numbers) {
+        this(List.of(new Lotto(numbers)));
+    }
+
+    public BuyLotto(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public List<Lotto> getLottos() {
+        return lottos;
+    }
+
+    public LottoResult match(Lotto winningLotto, NumberElement bonusNumber) {
```

Comment:
```suggestion
    public LottoResult match(WinningLotto winningLotto) {
```
클린 코드 책에서 제시하는 가이드
- 함수에서 이상적인 인수 개수는 0개(무항)이다. 다음은 1개이고, 다음은 2개이다.
- 3개는 가능한 피하는 편이 좋다.
- 4개 이상은 특별한 이유가 있어도 사용하면 안된다